server/apps/prepopulate/app_initialization_test.py

```diff
@@ -47,8 +47,6 @@ def data_scaffolding_test(self):
         self.assertEqual(cursor.count(), existing_desks * stories_per_desk)
 
     def test_app_initialization_index_creation(self):
-        result = app.data.mongo.pymongo(resource='users').db['users'].index_information()
```

Comment:
K

On Wed, Sep 23, 2015 at 2:23 PM Peter Lamut <notifications@github.com>
wrote:

> In server/apps/prepopulate/app_initialization_test.py
> <https://github.com/superdesk/superdesk/pull/1130#discussion_r40192548>:
>
> > @@ -47,8 +47,6 @@ def data_scaffolding_test(self):
> >          self.assertEqual(cursor.count(), existing_desks * stories_per_desk)
> >
> >      def test_app_initialization_index_creation(self):
> > -        result = app.data.mongo.pymongo(resource='users').db['users'].index_information()
>
> This was causing the test to incorrectly fail in Mongo 3.0.6 (did not test
> other versions FWIW). After a discussion with @mdhaman
> <https://github.com/mdhaman> (the author of this test) we determined that
> this is most likely due to a changed Mongo behavior. It used to return an
> empty dictionary if a collection did not exist ('users' in this case),
> but in 3.0.6 it returns None and calling a method on it fails of course.
>
> @mdhaman <https://github.com/mdhaman> then said that this line is not
> actually needed for the test and suggested to simply remove it, since the
> essential part of the test lies a few lines below.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/superdesk/superdesk/pull/1130/files#r40192548>.
>